internal/imports/v2/imports.go

```diff
@@ -50,8 +50,8 @@ func HostFunctions(instance common.WasmInstance) map[string]interface{} {
 
 	hostFuncs["proxy_get_header_map_pairs"] = h.ProxyGetHeaderMapPairs
 
-	hostFuncs["proxy_get_map_values"] = h.ProxyGetMapValues
-	hostFuncs["proxy_set_map_values"] = h.ProxySetMapValues
+	hostFuncs["proxy_get_header_map_value"] = h.ProxyGetHeaderMapValue
```

Comment:
there are other incompatibilities, but test data doesn't hit them yet. This starts the process of fixing the v2 ABI with real data.